FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -63,22 +65,23 @@ internal FirebaseUserManager(Args args)
                     DeserializeExceptionHandler = AuthErrorHandler.Instance,
                     RetryOptions = args.RetryOptions,
                 });
-            this.baseUrl = string.Format(IdTooklitUrl, args.ProjectId);
+            this.baseUrl = string.Format(args.TenantId != null ? IdTooklitTenantUrl : IdTooklitUrl, args.ProjectId, args.TenantId);
```

Comment:
Expand this out for readability.
```suggestion
            if (!string.IsNullOrEmpty(args.TenantId)
            {
                this.baseUrl = string.Format(IdTooklitTenantUrl, args.ProjectId, args.TenantId);
            }
            else
            {
                this.baseUrl = string.Format(IdTooklitUrl, args.ProjectId);
            }
```